### PR TITLE
Modernize the rest of the `index.html` files to support WASM compilation

### PR DESCRIPTION
**Pull request by Rexios80**
Turns out there are a bunch of other `index.html` files that used a different method of Flutter initialization

https://github.com/flutter/flutter/issues/151663

### CODE REVIEW

File: packages/google_identity_services_web/README.md
Comment (Rexios80) [inline packages/google_identity_services_web/README.md:27]: I wasn't 100% sure what to do here. Is this correct?

**Comment by stuartmorgan** [comment]
test-exempt: configuration change

CHANGELOG override: mechanical change to example plumbing to bring it up to current standard.

Note that you will need a version and CHANGELOG change where you have changed the README.

File: packages/google_identity_services_web/README.md
Comment (ditman) [reply packages/google_identity_services_web/README.md:27]: I'm not sure because I haven't tried this new index, but leaving it in the `head` wouldn't hurt. This needs to be tested, however (I suspect google maps had a similar change)

Comment (Rexios80) [reply packages/google_identity_services_web/README.md:27]: I actually get this warning in the Chrome dev console for google maps now. I tried using the script they give but it doesn't work.

```console
Google Maps JavaScript API has been loaded directly without loading=async. This can result in suboptimal performance. For best-practice loading patterns please see https://goo.gle/js-api-loading
```

Comment (Rexios80) [reply packages/google_identity_services_web/README.md:27]: Ah I see https://github.com/flutter/flutter/issues/132150 tracking that

**Review by ditman — APPROVED** [review]
LGTM! If web CI passes this should be good to go!

**Comment by ditman** [comment]
Ouch! I need another approval /cc @stuartmorgan

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by stuartmorgan** [comment]
@Rexios80 Please feel free to ping me on Discord if you would like to be nominated for contributor access!

**Pull request merged**
